### PR TITLE
fix z-index issues in demos

### PR DESCRIPTION
**Pull request by mayank99**
## Changes

changed `portalContainer` of all examples to `<body>`. fixes some website demos where content was getting clipped because of z-index.

## Testing

before:

image

image

after:

image

image

## Docs

n/a

### CODE REVIEW

**Review by gretanausedaite — APPROVED** [review]

File: examples/index.tsx [hunk 1]
```diff
@@ -6,8 +6,15 @@ import * as React from 'react';
 import { ThemeProvider } from '@itwin/itwinui-react';
 
 const withThemeProvider = (Component: () => React.ReactElement) => () => {
+  const [portalContainer, setPortalContainer] = React.useState<HTMLElement>();
+  React.useEffect(() => void setPortalContainer(document.body), []);
```
Comment (rohan-kadkol) [inline examples/index.tsx:10]: Is there a reason to not initialize the `portalContainer` itself? Like why not pass `document.body` to `useState()` on L9?

Does this have something to do with `document.body` not being available before `React.useEffect()` can be called? Like some SSR related quirk?

Comment (rohan-kadkol) [inline examples/index.tsx:10]: Why is `void` needed?

Comment (mayank99) [reply examples/index.tsx:10]: yes, `document` only works on the client

Comment (mayank99) [reply examples/index.tsx:10]: the `void` is to ensure that if there is ever a return a value, it will be discarded.

**Pull request merged**